app/controllers/sign_up/completions_controller.rb

```diff
@@ -86,5 +86,9 @@ def pii
       pii_string = Pii::Cacher.new(current_user, user_session).fetch_string
       JSON.parse(pii_string || '{}', symbolize_names: true)
     end
+
+    def multiple_factors_enabled?
+      return if MfaPolicy.new(current_user).multiple_factors_enabled?
+    end
```

Comment:
```suggestion
```

It looks like the view calls this method directly so this definition isn't needed?

Reply:
You are right. Fixing this now